.gitignore

```diff
@@ -16,3 +18,12 @@ profiles.clj
 dev/user.clj
 .lein-repl-history
 ext/packaging
+
+# Bundler local state files
+Gemfile.lock
+Gemfile.local
+/.bundle/
```

Comment:
They're not necessary, but they're more specific in that the directory should really only be excluded from the root of the project, not sub-directories.